classes/person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :@name, :@age
+  attr_reader :@id
```

Comment:
- [ ] Defining your getter and setter this way returns errors. This is what you should have:

  ```
  attr_accessor :name, :age
  attr_reader :id
  ```

Reply:
thank you so much @julie-ify